packages/api-explorer/src/Params.jsx

```diff
@@ -48,31 +55,35 @@ function Params({
         <Form
           key={`${schema.type}-form`}
           fields={{
-            DescriptionField,
             ArrayField,
+            DescriptionField,
             SchemaField,
           }}
           formData={formData[schema.type]}
-          id={`form-${operation.operationId}`}
-          idPrefix={operation.operationId}
+          id={`form-${schema.type}-${operationId}`}
```

Comment:
Deploying this will break existing links that may be currently shared around the web. Can we add a dummy div with the old id format so it is backwards compatible?

Reply:
Re-added as a `<div>` that's wrapping the entire component!